htdocs/newindex.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
note: having a file named "index.php" seems to be enough to make the php built in web server use that instead of the htdocs/router.php passed on the command line for routing.

This won't be a problem when we switch everything over in the next PR, but for this PR it needs to be called something other than "index.php" to pass Travis.